Lib/test/libregrtest/run_workers.py

```diff
@@ -294,16 +294,17 @@ def read_stdout(self, stdout_file: TextIO) -> str:
 
     def read_json(self, json_file: JsonFile, json_tmpfile: TextIO | None,
                   stdout: str) -> tuple[TestResult, str]:
+        worker_json: StrJSON
         try:
             if json_tmpfile is not None:
                 json_tmpfile.seek(0)
-                worker_json: StrJSON = json_tmpfile.read()
+                worker_json = json_tmpfile.read()
             elif json_file.file_type == JsonFileType.STDOUT:
                 stdout, _, worker_json = stdout.rpartition("\n")
                 stdout = stdout.rstrip()
             else:
                 with json_file.open(encoding='utf8') as json_fp:
-                    worker_json: StrJSON = json_fp.read()
+                    worker_json = json_fp.read()
```

Comment:
Maybe StrJSON type is not really helpful, and just str is better?

Reply:
Yeah, I think so

Reply:
Done in [ad6b96a](https://github.com/python/cpython/pull/109405/commits/ad6b96ae001601689b5a24f6ad68fc7d42604a27)